### DISCUSSION TITLE
Editor App to generate boilerplate for applications

### DISCUSSION THREAD

**Discussion by coding-beagle** [TomSchimansky/CustomTkinter#2268] [Showcase]
My attempt at making a VB inspired editor: feel free to check it out. Some interesting implementation includes dynamic generation of a top level element from a dict of field names, data types and extra flags (can be found within preferenceshandler.py and settingseditor.py). I'd be happy to refactor this into a general widget if anyone finds it useful.

[Repo link](https://github.com/coding-beagle/CTKEasyEditor), including installer and .zip for windows

![image](https://github.com/TomSchimansky/CustomTkinter/assets/87939112/9123274f-9155-41cb-a5c0-d112452591b3)

**Comment by Anonymous6598** [comment]
Great idea!

**Comment by Dumbliidore** [comment]
Great!
